### ISSUE TITLE
[Multiple Suggestions] Tiered accounts / limits / expiration

### ISSUE THREAD

**Issue by ghost** [chibisafe/chibisafe#39]
This is just a suggestion and it's specific to _my wants_, so it's probably not something that would be seen as a necessity. 

I have limited space on my vps with multiple websites, backups, etc.

I want to open it to the public, but I don't want them to have 100mb upload limit, I want it at 10mb for non registered users, but 100mb for registered users. 

For people with accounts, I don't want to delete their files, but I don't want public uploads to stay forever. I want it to expire after x amount of days ("pubExpire: 30" // public files expire after 30 days variable). (I think this is a request already?)

People with accounts I want them to be able to keep their uploads permanent but have a limit on how much they can upload. So registered users can have permanent uploads for say 5gb storage ("quota: 5GB").

Basically my suggestions are:

1. Public non registered max upload size
2. Registered max upload size
3. Account storage quotas

Also while I'm here, does anyone have a script I can have to add accounts to database so I don't have to change allow registrations in config and restart the service? Or any other scripts (deleting after x amount of days).

Thanks :)

**Comment by Pitu** [comment]
Thanks for the suggestion! 
I'm making a public kanboard soon to organize the new features lolisafe is gonna have on it's next version, a few weeks from now. Maybe I can squeeze in the storage quota as well.

**Comment by alucard0134** [comment]
this goes completely against the pomf design idea, and this thought deserves to die

**Comment by Pitu** [comment]
Good thing we are not a pomf clone.

**Comment by notstevek** [comment]
Sorry :( just something I would have liked to see from having multiple users on a small vps. Was not aware of specific guidelines. Repo owner can close this if they'd like.

**Comment by Pitu** [comment]
No @notstevek , I believe those would be nice suggestions. Will keep them in mind, I'm not following those old pomf guidelines.

**Comment by notstevek** [comment]
@Pitu I am happy with how this is now and none of the other pomf like upload projects have met my needs as much as this has. Like I said in the op, this was just a suggestion based on my wants. If the community disagrees with it, I would not want this to cause any issues. Anyway, thanks for the response and for your work.

**Comment by Pitu** [comment]
Pinging @BobbyWibowo for this one 🙈
